codec-http2/src/main/java/io/netty/handler/codec/http2/Http2FlowController.java

```diff
@@ -28,7 +28,7 @@
      * @param ctx The {@link ChannelHandlerContext} for which to apply flow control on.
      * @throws Http2Exception if any protocol-related error occurred.
      */
-    void channelHandlerContext(ChannelHandlerContext ctx) throws Http2Exception;
+    T channelHandlerContext(ChannelHandlerContext ctx) throws Http2Exception;
```

Comment:
Just using the builder pattern for setters. This is consistent with other places in Netty and changing this to be consistent with the new setter for the frameWriter.